### ISSUE TITLE
It would be nice if package tutorials could be listed like vignettes in packages documentation.

### ISSUE THREAD

**Issue by rpruim** [rstudio/learnr#91]
Is this controlled by core R or is it possible to add tutorials to the "User guides, package vignettes and other documentation." that shows up when you click on a package name in the Packages tab.

**Comment by jjallaire** [comment]
Yes, that's part of R itself. We are considering adding a tutorial/vignette
browser to RStudio but work on that would still be a ways off.

**Comment by wlandau-lilly** [comment]
Would that include a way to automatically view vignettes as tutorials? [One of my packages](https://github.com/wlandau-lilly/drake) has [8 vignettes](https://github.com/wlandau-lilly/drake/tree/master/vignettes). I need to stay under the 5 MB storage limit, and I would like to avoid duplicated effort.
